src/test/java/org/llorllale/cactoos/matchers/HasEntryTest.java

```diff
@@ -35,10 +35,6 @@
  * Test case for {@link HasEntry}.
  *
  * @since 1.0.0
- * @todo #65:30min Add two more tests for the mismatch message,
```

Comment:
@victornoel just pointing out that the request was of two more tests, while the edit seems to do work also for #136. Maybe in the end the tests are just fine, but separate tests for the functionalities and the mismatch message could be more readable, what do you think? I know it's all part of issues you're working on, just trying to review as critically as possible :)

Reply:
@scristalli I don't see how it is possible to test the mismatch messages with `Mismatches` without testing the functionality at the same time. And I don't see how I can test that a `Matcher` mismatches without testing the messages at the same time. If you see how this could be done without just duplicating tests and for the sake of having more tests, I can do it though :)

I believe the todo is misleading because at the time, there was no `Mismatches` so we didn't know it could be so "powerful" to test the behaviour of a `Matcher` at once :)

Reply:
@victornoel I think you are right, in this case there would be duplication in tests. I was thinking of minimal code just for testing messages, but this is precisely what we already have, no need to improve.